apps/passport-server/migrations/37_add_encryption_key_column.sql

```diff
@@ -0,0 +1,5 @@
+-- Add column for storing a user's encryption key for their data,
```

Comment:
This file will conflict with my PR #803.  Whoever merges last will have to renumber the file, but that should be all.